aws-java-sdk-core/src/main/java/com/amazonaws/retry/PredefinedRetryPolicies.java

```diff
@@ -217,7 +221,7 @@ public final long delayBeforeNextRetry(AmazonWebServiceRequest originalRequest,
                 scaleFactor = SCALE_FACTOR;
             }
 
-            long delay = (1 << retriesAttempted) * scaleFactor;
```

Comment:
The change of bit-wise shift from 1 to 1L also looks unnecessary.  This increase the runtime overhead, albeit very small, with no advantages.  What am I missing ?